test/tests.c

```diff
@@ -297,10 +297,10 @@ void check_store_file(int error_code, char *file_id, void *handle)
         printf("\t\tERROR:   %s\n", storj_strerror(error_code));
     }
 
-    free(file_id);
+    storj_free_uploaded_file_info(file);
```

Comment:
check if file is NULL

Reply:
The `storj_free_uploaded_file_info()` function does this check. So it should not be necessary here, right?